root/main/error/components/ErrorEnvironment.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * @flow strict
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+type Props = {
+  +$c: CatalystContextT,
+  +hostname?: string,
+  +useLanguages?: boolean,
+};
+
+const ErrorEnvironment = ({
```

Comment:
Commit https://github.com/metabrainz/musicbrainz-server/pull/1699/commits/faf46ccac63eaa0d97b71c59dc05a63cdcb418e3 gives no indication of what this is being converted from.

Reply:
Fair enough, added the info to the commit message

Reply:
Thank you!